doc/developers.rst

```diff
@@ -451,6 +444,138 @@ DataAccess class i.e. the `_field_write_arguments()` and
 `Arguments` class.
 
 
+Parsing Code
+############
+
+The PSyclone `parse` module is responsible for parsing science
+(algorithm and kernel) code and extracting the required information
+for the algorithm translation and PSy generation phases.
+
+The `parse` module contains modules for parsing algorithm
+(`algorithm.py`) and kernel (`kernel.py`) code as well as a utility
+module (`utils.py`) for common functionality.
+
+Parsing Algorithm Code
+======================
+
+The first thing PSyclone typically does is parse an input file. The
```

Comment:
Please clarify that by "input file" you mean a Fortran source file containing a PSyKAl algorithm definition.

Reply:
Clarified.